app/controllers/profile.js

```diff
@@ -53,12 +54,21 @@ module.exports = {
         constraints.where["data.country.label"] = req.body.country;
       // city: either it will be an array or a single value
       // array with array
-      if (req.body.city)
-        constraints.where["data"] = {
-          [Op.contains]: {
-            city: [{ label: req.body.city, value: req.body.city }],
-          },
-        };
+      if (req.body.type === "fm-buyer") {
+        if (req.body.city)
+          constraints.where["data"] = {
```

Comment:
@ChaitanyaV98  no need to use contain here, 
`constraints.where["data.city.label"] = req.body.city;`